projects/igniteui-angular/src/lib/grids/selection/selection.service.ts

```diff
@@ -139,11 +140,9 @@ export class IgxGridCRUDService {
             console.warn('The grid must have a `primaryKey` specified when using `rowEditable`!');
         }
         this.row = this.createRow(this.cell);
-        const args = {
-            rowID: this.row.id,
-            oldValue: this.row.data,
-            cancel: false
-        };
+        // TODO: should we emit newValue in onRowEditEnter?
```

Comment:
Why Cell/Row edit enter events need to have a property newValue since no edit action is performed?!